service/ibmmq-plugin/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.solace.maas</groupId>
+        <artifactId>maas-event-management-agent-parent</artifactId>
+        <version>1.3.2-SNAPSHOT</version>
```

Comment:
please update this with latest version before merging, we are currently at version 1.4.2 and will soon release 1.5.0